src/gpu-compute/GPU.py

```diff
@@ -96,6 +96,14 @@ class VectorRegisterFile(RegisterFile):
     cxx_header = "gpu-compute/vector_register_file.hh"
 
 
+class RegisterFileCache(SimObject):
+    type = "RegisterFileCache"
+    cxx_class = "gem5::RegisterFileCache"
+    cxx_header = "gpu-compute/register_file_cache.hh"
+    simd_id = Param.Int(-1, "SIMD ID associated with this Register File Cache")
+    cache_size = Param.Int(0, "capacity of rfc")
```

Comment:
capacity means size (number of entries), right?  If so maybe we should update description.